docs/HowTo/Builder-Endpoint.md

```diff
@@ -0,0 +1,157 @@
+---
+description: How to connect to a builder endpoint
+---
+
+# Connect to a builder endpoint
+
+You can connect a [builder endpoint](../Concepts/Builder-Endpoint.md) to generate execution
+payloads for the [consensus client](../Concepts/Merge.md#consensus-clients).
+
+New blocks are recommended by the builder and validated by consensus client.
+You can [configure an endpoint](#configure-an-endpoint) and [monitor logging](#monitor-logging).
+
+## Configure an endpoint
+
+To configure a builder endpoint, specify the following options:
+
+* [enable blinded block production](#enable-blinded-block-production)
+* [specify the builder endpoint](#specify-the-builder-endpoint)
+* [register the validator](#register-the-validator)
+
+### Enable blinded block production
+
+Enable blinded block production using the command line option [`--validators-proposer-blinded-blocks-enabled`](../Reference/CLI/CLI-Syntax.md#validators-proposer-blinded-blocks-enabled)
+
+!!! note
+
+    If [--validators-builder-registration-default-enabled](../Reference/CLI/CLI-Syntax.md#validators-builder-registration-default-enabled) is set to true, then `--validators-proposer-blinded-blocks-enabled` is automatically enabled, so you don't need to set it explicitly.
+
+### Specify the builder endpoint
+
+Specify the builder endpoint using [`--builder-endpoint`](../Reference/CLI/CLI-Syntax.md#builder-endpoint)
+
+```bash
+--builder-endpoint="https://builder-relay-sepolia.flashbots.net/"
+```
+
+The following builder relay endpoints are live and can be used for testing.
```

Comment:
I would replace the table with reference to https://github.com/flashbots/mev-boost page which contains urls for different networks since they are prone to change